frontend/src/CreateAnnotationCampaign.js

```diff
@@ -342,6 +342,13 @@ class CreateAnnotationCampaign extends Component<CACProps, CACState> {
     const datasetOptions = utils.objectValues(this.state.dataset_choices).map(dataset => (
       <option key={`dataset-${dataset.id}`} value={dataset.id.toString()}>{dataset.name}</option>
     ));
+    let wanted_annotions_label = "";
```

Comment:
There's a typo in the variable name. I guess `wanted_annotators_label` will be ok?